<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UFOHunting</class>
 <widget class="QMainWindow" name="UFOHunting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(202, 255, 204)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>831</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(99, 175, 104)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>841</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>841</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(63, 114, 70)</string>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 14px;
border-width: 0px;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Alpha Shape</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 14px;
border-width: 0px;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Convex Hull</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 14px;</string>
     </property>
     <item>
      <property name="text">
       <string>-Select-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cigar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>circle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cylinder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>diamond</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>disk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>egg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>oval</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sphere</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>teardrop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>triangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rectangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>changing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fireball</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>formation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>flash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>light</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>other</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unknown</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 14px;</string>
     </property>
     <item>
      <property name="text">
       <string>-Select-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2012</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2011</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2010</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2009</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2008</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2007</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2006</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2005</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2004</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2003</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2002</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1999</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1998</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1997</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1996</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1995</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1994</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1993</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1992</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1991</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1990</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1989</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1988</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1987</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1986</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1985</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1984</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1983</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1982</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1981</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1980</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1979</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1978</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1977</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1976</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1975</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1974</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1973</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1972</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1971</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1970</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1969</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1968</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1967</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1966</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1965</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1964</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1963</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1962</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1961</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1960</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1959</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1958</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1957</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1956</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1955</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1954</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1953</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1952</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1951</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1950</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1949</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1948</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1947</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1946</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1945</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1944</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1943</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1942</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1941</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1940</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1910</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 14px;
border-width: 0px;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 16px;
border-width: 0px;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Shape:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 16px;
border-width: 0px;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
    <zorder>radioButton_2</zorder>
    <zorder>radioButton</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_2</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_3</zorder>
    <zorder>comboBox_2</zorder>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>791</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>851</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(63, 114, 70);
color: rgb(0, 0, 0);
font-family: OCR A Extended;
font-size: 40px;
border-width: 0px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>             UFO Map</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>label</zorder>
   <zorder>customPlot</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
